extensions/amp-bind/0.1/test/test-bind-impl.js

```diff
@@ -392,4 +422,57 @@ describes.realWin('Bind', {
           sinon.match(/Maximum number of bindings reached/));
     });
   });
+
+  describes.realWin('in embedded window', {
+    amp: {
+      runtimeOn: false,
+    },
+  }, otherEnv => {
+    let otherBind;
+    let otherParent;
+
+    let otherElementWithBinding;
+    let onOtherBindReady;
+    let onOtherBindReadyAndSetState;
+
+    beforeEach(() => {
+      otherBind = new Bind(env.ampdoc, otherEnv.win);
+      otherParent = otherEnv.win.document.getElementById('parent');
+
+      // TODO(choumx): Better names in a follow-up PR.
+      otherElementWithBinding =
+          createElementWithBinding_(otherParent, otherEnv);
+      onOtherBindReady = onBindReady_(otherBind, otherEnv);
+      onOtherBindReadyAndSetState =
+          onBindReadyAndSetState_(otherBind, otherEnv);
+    });
+
+    it('should scan the elements in the provided window', () => {
```

Comment:
Optional:
`should scan the elements in the provided window only` ? 
`should scan the elements only in the provided window` ?